browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3967,22 +3967,23 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_ALWAYS_ON_VPN" desc="Vpn setting title for always on">
       Always-on VPN
     </message>
-    <message name="IDS_HOW_TO_TURN_ON" desc="Vpn kill swicth tutorial text">
+    <message name="IDS_HOW_TO_TURN_ON" desc="Vpn kill switch tutorial text">
       How to turn on:
     </message>
-    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_1" desc="Vpn kill swicth tutorial text">
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_1" desc="Vpn kill switch tutorial text">
       1. Open your device's settings from your app drawer or tapping the button below.
     </message>
-    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_2" desc="Vpn kill swicth tutorial text">
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_2" desc="Vpn kill switch tutorial text">
       2. Find Brave VPN and tap on the settings icon to the right.
     </message>
-    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_3" desc="Vpn kill swicth tutorial text">
+    <message name="IDS_KILL_SWITCH_TUTORIAL_TEXT_3" desc="Vpn kill switch tutorial text">
       3. Toggle <ph name="ALWAYS_ON_TUTORIAL_START">&lt;always_on_tutorial&gt;</ph>Always-on VPN<ph name="ALWAYS_ON_TUTORIAL_END">&lt;/always_on_tutorial&gt;</ph> to enable it.
     </message>
-    <message name="IDS_KILL_SWITCH_ACTION_TEXT" desc="Vpn kill swicth tutorial action text">
+    <message name="IDS_KILL_SWITCH_ACTION_TEXT" desc="Vpn kill switch tutorial action text">
       Open Android settings
     </message>
-    <message name="IDS_IN_APP_UPDATE_TEXT" desc="Text for in app update snackbar">
+    <message name="IDS_IN_APP_UPDATE_TEXT" desc="Text for in git diff
+      app update snackbar">
```

Comment:
Looks like an accidental paste of `git diff` in here.

Reply:
oops. good catch